Makefile

```diff
@@ -46,16 +46,19 @@ requirements: node_requirements python_requirements ## install development envir
 install_linux_dev_firefox: ## Downloads custom version of firefox for Selenium in Linux
 	@echo "This works only on Linux. For MacOS please check the README file"
 
-	rm -rf .firefox
+	rm -rf .firefox && rm -rf .geckodriver
 	mkdir .firefox
+	mkdir .geckodriver
```

Comment:
Nit: if we're removing these directories in one line, we could also create them in a similar way (`mkdir .firefox .geckodriver`).

Reply:
Done